public/editor/scripts/editor/js/fc/publisher.js

```diff
@@ -30,7 +30,8 @@ define(function(require) {
         publish: $("#publish-button-publish"),
         update: $("#publish-button-update"),
         unpublish: $("#publish-button-unpublish"),
-        parent: $("#publish-buttons")
+        parent: $("#publish-buttons"),
+        index_message: $("#no-index")
```

Comment:
Is `index_message` your addition, or was it already in our tree?  In JS, we favour `thisStyle` vs. `this_style` (the latter is more common in Python).